hpx/parallel/algorithms/sort.hpp

```diff
@@ -201,19 +202,22 @@ namespace hpx { namespace parallel { inline namespace v1
                 result = execution::async_execute(policy.executor(),
                         &sort_thread<ExPolicy, RandomIt, Compare>,
                         std::ref(policy), first, last, comp);
+
+                return result;
```

Comment:
Wouldn't a simple

    return execution::async_execute(policy.executor(),  
        &sort_thread<ExPolicy, RandomIt, Compare>,  
        std::ref(policy), first, last, comp);

do the trick here?

Reply:
@hkaiser You're right. That code was derived from just moving ```return result;``` into inside of try block. I'll fix that like you said.